src/debian/stretch/docker-testrunner/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Dockerfile used to create a testrunner image that can perform Docker operations.
+#     docker run --rm -v /var/run/docker.sock:/var/run/docker.sock testrunner pwsh -File xyz.ps1
```

Comment:
Minor - `#     docker` extra space in between can be removed

Reply:
Replaced with `Usage:  `